cmd/mimir/help-all.txt.tmpl

```diff
@@ -2987,6 +2987,12 @@ Usage of ./cmd/mimir/mimir:
     	Override the default minimum TLS version. Allowed values: VersionTLS10, VersionTLS11, VersionTLS12, VersionTLS13
   -ruler.query-frontend.grpc-client-config.tls-server-name string
     	Override the expected name on the server certificate.
+  -ruler.query-frontend.max-retries int
+    	[experimental] Maximum number of retries for a single request. (default 3)
```

Comment:
this is actually max attempts, right? So it includes the first try. That's my understanding from reading the dskit code. I think we should clarify the CLI flag then.